src/layouts/TabLayout/TabLayout.js

```diff
@@ -27,17 +27,24 @@ export const TabLayout = (props) => {
   const { t } = useTranslation();
   const location = useLocation();
   const currentTabPathname = location?.pathname;
-  const scenarioViewUrl = useMatch('/scenario/:id');
+  const scenarioViewUrl = useMatch(':workspaceId/scenario/:scenarioId');
+  const instanceViewUrl = useMatch(':workspaceId/instance/:scenarioId');
   const applicationError = useApplicationError();
   const clearApplicationErrorMessage = useClearApplicationErrorMessage();
+  const routerParameters = useParams();
+  sessionStorage.removeItem('previousURL');
```

Comment:
I can't find where `sessionStorage` is declared in the diff, is the import present ?